vulnerabilities/models.py

```diff
@@ -188,13 +196,13 @@ def __str__(self):
 
 class PackageRelatedVulnerability(models.Model):
 
-    package = models.ForeignKey(
-        Package, on_delete=models.CASCADE, related_name="vulnerable_package"
-    )
+    package = models.ForeignKey(Package, on_delete=models.CASCADE, related_name="package")
     vulnerability = models.ForeignKey(Vulnerability, on_delete=models.CASCADE)
-    patched_package = models.ForeignKey(
-        Package, on_delete=models.CASCADE, null=True, blank=True, related_name="patched_package"
+    source = models.TextField(null=True)
```

Comment:
Please add help to each field. See below. Also we may want to have a charfield instead?
Please also check the use of blank and null... to be consistent with other Text fields

Reply:
Added and `blank`ed